pkgs/racket-doc/scribblings/reference/match.scrbl

```diff
@@ -662,17 +662,20 @@ many values to expect from @racket[expr].
   ]
 }
 
-@defform[(match-lambda clause ...)]{
+@deftogether[(@defform[(match-lambda clause ...)]
+              @defform[(match-λ clause ...)])]{
 
 Equivalent to @racket[(lambda (id) (match id clause ...))].
 }
```

Comment:
Please add `@history[#:changed "8.13.0.5" @elem{Added @racket[match-λ].}]` here before the closing curly brace, and similar for the other two. I'll bump the Racket version number after merging this and related PRs.